test/functional/testlib.bash

```diff
@@ -2928,6 +2924,7 @@ print_assert_failure() {
 	echo -e "$message"
 	echo "Command output:"
 	echo "------------------------------------------------------------------"
+	# shellcheck disable=SC2154
```

Comment:
add a comment why you are disabling this checker

Reply:
done